docs/elements.md

```diff
@@ -1255,6 +1256,45 @@ Note that some Python types may not be serialized correctly in the code provided
 - [`pl-variable-output` for displaying a matrix or element in code form.](#pl-variable-output-element)
 - [`pl-python-variable` for displaying a formatted output of Python variables.](#pl-python-variable-element)
 
+### `pl-details` element
+
+Displays question content within a collapsible accordion-style component.
+
+![pl-details element closed](elements/pl-details-closed.png)
+![pl-details element opened](elements/pl-details-open.png)
+
+#### Sample element
+
+```html title="question.html"
+<pl-question-panel>
+  <pl-details title="Large table of data">
+    <pl-dataframe params-name="df"></pl-dataframe>
```

Comment:
I think this example may be misconstrued as it requiring a custom element as its inner body. I think having a simpler example here using HTML only may make it clearer what it's intended to use. Alternatively, add a caption or similar text before or after the dataframe.

Reply:
Changed in https://github.com/PrairieLearn/PrairieLearn/commit/6199a64fd30c84b3a4e3b86ab320389b6446d929 to the content in the screenshot.